swagger.yml

```diff
@@ -138,44 +151,44 @@ paths:
         "204":
           description: "No Content"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+          $ref: "#/responses/UnauthenticatedError"
         "403":
-          $ref: '#/responses/UnauthorizedError'
+          $ref: "#/responses/UnauthorizedError"
         "503":
-          $ref: '#/responses/InternalServerError'
+          $ref: "#/responses/InternalServerError"
       security:
         - basicAuth: []
   /wishlist:
     post:
       tags:
-      - "wishlist"
+        - "wishlist"
       summary: "Create a wishlist"
       description: "This can be done only by an authenticated user"
       operationId: "wishlist"
       consumes:
-      - "application/json"
+        - "application/json"
       produces:
-      - "application/json"
+        - "application/json"
       parameters:
-      - in: "body"
-        name: "body"
-        description: "Wishlist items"
-        required: true
-        schema:
-          $ref: "#/definitions/wishlist"
+        - in: "body"
+          name: "body"
+          description: "Wishlist items"
+          required: true
+          schema:
+            $ref: "#/definitions/wishlist"
       responses:
         "201":
           description: "Item successfully added"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+          $ref: "#/responses/UnauthenticatedError"
         "403":
-          $ref: '#/responses/UnauthorizedError'
+          $ref: "#/responses/UnauthorizedError"
         "405":
           description: "Method not allowed"
         "503":
-          $ref: '#/responses/InternalServerError'
+          $ref: "#/responses/InternalServerError"
```

Comment:
Any difference in between ' and " ?

Reply:
' gets changed to " because of code formatter.